bin/asini.js

```diff
@@ -39,6 +40,8 @@ var cli = meow([
   "  --repo-version       Specify repo version to publish",
   "  --concurrency        How many threads to use if asini parallelises the tasks (defaults to 4)",
   "  --loglevel           What level of logs to report (defaults to \"info\").  On failure, all logs are written to asini-debug.log in the current working directory.",
+  "  --dry                Don't eject any packages, only log where they would be moved",
```

Comment:
I renamed it dry-run, and clarified that its _currently_ only supported by export -- lemme know if you have improvements to the message; I wanted to keep it tight, but I think I might have been too brief.